scripts/new_version

```diff
@@ -4,13 +4,23 @@
 Make a new release version of BiblioPixel by editing CHANGELIST.md
 
 """
+# TODO: Migrate to https://github.com/olivierlacan/keep-a-changelog
```

Comment:
I know we have existing TODOs in the code, but that's because we are slack.  :-)   TODOs in code are a form of technical debt that you cannot track - and that there is no way to discuss without polluting the code.  Usually we use issues for this purpose...

Reply:
Yeah, this was more of an internal note as I develop more on this PR and some changes/additions I wanted to propose.

Due to the size of this project, looking into project trackers would significantly benefit development. I briefly mentioned services like Pivotal Tracker, ZenHub, and Jira.
There are so many out there, but a majority of them are free for open-source projects, and would help in roadmapping, handling issues, and tracking development and fixes.